common/persistence/visibility/store/standard/cassandra/visibility_store.go

```diff
@@ -143,7 +143,7 @@ func NewVisibilityStore(
 
 	return &visibilityStore{
 		session:      session,
-		lowConslevel: gocql.One,
+		lowConslevel: gocql.Consistency(cfg.Consistency.GetConsistency()),
```

Comment:
@dhruva-groww, is there specific issue that you are encountered with this?
The consistency level One is used intentionally as lowConslevel for read operations. I'm not sure why the Delete operation is also using this lowConslevel, is it the one that is causing trouble? 
cc @alexshtin @rodrigozhou

Reply:
Yes. I assumed the writes that occur should occur as per the configuration provided in config (default as local_quorum) under visibility store.
I discovered this issue while integrating with AstraDB which does not allow ONE as a consistency level while writing (and in general may cause data loss). I had raised it here with logs and exceptions
https://community.temporal.io/t/getting-error-on-deleting-workflows-while-using-astradb-serverless-with-temporal/5356

Reply:
Instead of changing the lowConslevel here, could you change the DeleteWorkflow to not use consistency One? https://github.com/temporalio/temporal/blob/d56e1d6fe3430a8ef5afde65b1daec783893fbbe/common/persistence/visibility/store/standard/cassandra/visibility_store.go#L477

Reply:
On it

Reply:
Done